src/rp2040/boot_stage2/CMakeLists.txt

```diff
@@ -43,15 +43,16 @@ function(pico_define_boot_stage2 NAME SOURCES)
     if (PICO_C_COMPILER_IS_CLANG)
         target_link_options(${NAME} PRIVATE "-nostdlib")
     elseif (PICO_C_COMPILER_IS_GNU)
-        target_link_options(${NAME} PRIVATE "--specs=nosys.specs")
+        target_compile_options(${NAME} PRIVATE "--specs=picolibc.specs")
+        target_link_options(${NAME} PRIVATE "--specs=picolibc.specs")
         target_link_options(${NAME} PRIVATE "-nostartfiles")
     endif ()
 
     # boot2_helpers include dir
     target_include_directories(${NAME} PRIVATE ${PICO_BOOT_STAGE2_DIR}/asminclude)
 
     target_link_libraries(${NAME} hardware_regs boot_stage2_headers)
-    target_link_options(${NAME} PRIVATE "LINKER:--script=${PICO_BOOT_STAGE2_DIR}/boot_stage2.ld")
+    target_link_options(${NAME} PRIVATE "-T${PICO_BOOT_STAGE2_DIR}/boot_stage2.ld")
```

Comment:
It's a minor point, but I think long options are easier to read in scripts.

Reply:
picolibc's spec fragments only look for the -T version of this option, so we must use the short variant to prevent addition of the picolibc linker script here.

Reply:
OK, I got it.
Thank you for your clarification.